scoreCard.test.js

```diff
@@ -0,0 +1,15 @@
+const ScoreCard = require('./scoreCard');
+
+describe('ScoreCard class', () => {
+  const sc = new ScoreCard();
+
+  it ('prints the frame scores', () => {
+    expect(sc.printFrameScores()).toEqual("Frame: 1, First roll: 1, Second roll: 0, Bonus: none")
+  });
+
+  it('prints the total scores', () => {
```

Comment:
these tests don't seem to be passing for me. I'm getting undefined. I got a bit distracted trying to work out why. I wondered if it was something to do with when you're in the 9th and 10th frames, and trying to access index's which don't actually exist in the array you're calling them on?! I encountered an issue like this and it was resulting in undefined for me